src/style.css

```
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+  outline: none;
+}
+
+body {
+  margin: 20px 8px;
+  font-weight: 300;
+}
+
+.search {
+  position: relative;
+  margin-bottom: 20px;
+}
+
+.search object {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.searchbar {
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.searchbar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchbar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 900;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.search.type-1 .searchbar {
+  padding: 23px 10px 23px 62px;
+}
+
+.search.type-2 .searchbar {
```

if you add modifiers you will use like this:
```suggestion
.form__searchbar--big {
  padding: 23px 10px 23px 62px;
}

.form__searchbar--small {
```